src/Symfony/Component/HttpKernel/Tests/Fixtures/Controller/NullableController.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Symfony\Component\HttpKernel\Tests\Fixtures\Controller;
+
+class NullableController
+{
+    public function action(? string $foo, ? \stdClass $bar, ? string $baz = 'value', $mandatory)
```

Comment:
Sorry for my minor comment but I don't know if a syntax has been defined yet for this, because I don't really like having a space after the `?`. The [RFC](https://wiki.php.net/rfc/nullable_types) for now don't use any space.